patches/relay-compiler+7.1.0.patch

```diff
@@ -25,17 +25,14 @@ index d0ae8f5..3d47c54 100755
        sources.push(source.body);
        astDefinitions.push.apply(astDefinitions, (0, _toConsumableArray2["default"])(ast.definitions));
      });
-@@ -15352,9 +15352,10 @@ function find(tagFinder, text, absPath) {
+@@ -17478,9 +17478,7 @@ function find(tagFinder, text, absPath) {
```

Comment:
What do these two patches do and why are they needed?

Reply:
This relay-compiler patch is to give us 'clickable' file names with correct `line:column` numbers in error messages.

Reply:
@ds300 so to confirm, if the two patches in this PR were to suddenly go away it would have no effect on application behavior?

Reply:
right :+1:

Reply:
Although you would have to clear the relay-compiler cache in `$TMPDIR`